knora/dsplib/models/resource.py

```diff
@@ -394,7 +393,7 @@ def _get_baseclass(self, superclasses: list[str]) -> Union[str, None]:
             return self._get_baseclass(gaga.superclasses)
         return None
 
-    def get_resclass(self, prefixedresclass: str) -> Type:
+    def get_resclass_type(self, prefixedresclass: str) -> Type:
```

Comment:
What's the difference between a resource class and a resource class type?

Reply:
As far as I understand, there are resourceclass names (the parameter of this method, a string), and resourceclass types (the return value of this method, a https://docs.python.org/3.9/library/functions.html#type)